src/Symfony/Component/HtmlSanitizer/HtmlSanitizerAction.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer;
+
+enum HtmlSanitizerAction: string
```

Comment:
no need to make it backed?
```suggestion
enum HtmlSanitizerAction
```

Reply:
maybe keep it backed so it can easily be create from yaml config (from string to this enum)

Reply:
Yes that's why i made it backed. I figured it'd make the bundle changes/integration easier.